### DISCUSSION TITLE
Welcome to Fluss Discussions!

### DISCUSSION THREAD

**Discussion by wuchong** [apache/fluss#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by trullia** [comment]
很优秀的项目，实时数据的未来发展会越来越好～

**Comment by fengziboboy** [comment]
先star，看看怎么用上

**Comment by trullia** [comment]
Your email has been received, thank you very much for your mail, I will be related to reply to you as soon as possible.
